docs/terraformoptions.adoc

```diff
@@ -1000,14 +1006,39 @@ Refer to {uri-topology}[topology] for more thorough examples.
 |Default
 
 |enable_calico
-|Whether to install {uri-calico}[Calico] as {uri-calico-policy}[pod network policy].
+|Whether to install {uri-calico}[Calico] for pod {uri-calico-policy}[network policy] and/or {uri-calico-networking}[networking].
 |true/false
 |false
 
 |calico_version
 |Version of {uri-calico}[Calico] to install.
-|3.19
-|3.19
+|3.7-3.21, 3.24.1-3.24.5
+|3.24.5
+
+|calico_mode
+|{uri-calico}[Calico] installation {uri-calico-manifests}[mode].
+|canal, vxlan, flannel-migration, policy-only, ipip
```

Comment:
DNS doesn't work when mode=vxlan. 

I'm ok to document this as a known issue and merge until we can troubleshoot this.

Reply:
Good catch - I think I found the cause and am testing, otherwise I'll document as you're suggesting.